src/dragonball/src/event_manager.rs

```diff
@@ -63,6 +66,15 @@ impl Drop for EventManager {
             .ok();
     }
 }
+impl std::fmt::Debug for EventManager {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("EventManager")
+            .field("epoll_mgr", &self.epoll_mgr.type_id())
```

Comment:
Do we need to get the type_id of epoll mgr?

Reply:
Unnecessary, fixed.